src/DynamoCoreWpf/UI/GuidedTour/GuidesValidationMethods.cs

```diff
@@ -569,7 +569,7 @@ internal static void ExecuteViewDetailsSideBar(Step stepInfo, StepUIAutomation u
                 if (packageManagerSearchElementViewModel == null) return;
 
                 if (packageDetailsWindow == null)
-                    packageManagerViewModel.ViewPackageDetailsCommand.Execute(packageManagerSearchElementViewModel.Model);
+                    packageManagerViewModel.ViewPackageDetailsCommand.Execute(packageManagerSearchElementViewModel.SearchElementModel);
```

Comment:
whats the difference between the Model and SearchElementModel - I would definitely also have thought the model was the appropriate parameter for this command?

Does the type checking or documentation for the command need to be improved?

Reply:
There is no difference between Model and SearchElementModel. The name `Model` was causing a AmbiguousMatchException error, as it was hiding the `Model` of the base class, thus causing the ambiguity. Renaming the property solved that issue.